proposals/jorydotcom-PROJECT_PROGRESSION/PROJECT_PROGRESSION.md

```diff
@@ -0,0 +1,116 @@
+## I. Overview
+This governance policy describes how an open source project can formally join the Foundation via the [Project Proposal Process](). It describes the [Stages]() a project may be admitted under and what the criteria and expectations are for a given stage, as well as the acceptance criteria for a project to move from one stage to another. It also describes the [Annual Review Process]() through which those changes will be evaluated and made. 
+
+Benefits of the project progression process include:
+* Promotion and public visibility of work that can add value to the _$FOUNDATION_ mission and build the ingredients of a successful project
+* Deepened alignment with existing projects
+* Mentorship and support 
+* Remove possible legal and governance obstacles to adoption and contribution by ensuring all projects adhere to _$FOUNDATION_ legal, code of conduct and IP Policy requirements
+
+This proposal has been modified from the [CNCF process documentation](https://github.com/cncf/toc/tree/master/process).
+
+## II. Project Proposal Process
+
+### Introduction
+This governance policy sets forth the proposal process for projects to be accepted into the _$FOUNDATION_. The process is the same for both existing projects which seek to move into the _$FOUNDATION_, and new projects to be formed within the _$FOUNDATION_.
+
+### Project Proposal Requirements
+Projects must be proposed via GitHub. Project proposals submitted to the _$FOUNDATION_ must provide the following information to the best of their ability:
+
+* name of project
+* project description (what it does, why it is valuable, origin and history)
+* statement on alignment with _$FOUNDATION_ charter mission
```

Comment:
I would add here, 

* Link to current code of conduct.

Reply:
+1